core/src/test/java/nl/nn/adapterframework/xslt/XsltTestBase.java

```diff
@@ -283,7 +241,6 @@ public void documentIncludedInSourceAbsoluteXslt1() throws Exception {
 	@Test
 	public void documentIncludedInSourceAbsoluteXslt2() throws Exception {
 		setStyleSheetName("/Xslt/importDocument/importLookupAbsolute2.xsl");
-		setXslt2(false);
```

Comment:
Volgens mij is dit vooral om er voor te zorgen dat de tests sneller worden (omdat het framework niet hoef te spieken welke xslt version in de file gedefineerd staat). Hier is het handiger om `xsltVersion(1);` te gebruiken.